packages/google-drive-kit/src/board-server/server.ts

```diff
@@ -125,6 +125,7 @@ class GoogleDriveBoardServer
       configuration.url,
       async () => {
         await this.refreshProjectList();
+        this.dispatchEvent(new Event("bbgraphboardserverrefresh"));
```

Comment:
can be just "refreshprojectlist". Also, can you add it to the list of mapped events here:
https://github.com/breadboard-ai/breadboard/blob/840a88a6b267d207ad9450e8d063e389ba69c45f/packages/breadboard/src/loader/types.ts#L346

Reply:
do you mean to create to create a new type of event?